### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by jasmingacic**

### CODE REVIEW

**Comment by edevenport** [comment]
Can you update the create server example to include `user_ssh_keys` just so it's very explicit that these are separate? Specifically:

```
device = compute.devices.create(:project_id => project_id, :facility => "ewr1", :plan => "baremetal_0", :hostname => "test01", :operating_system => "coreos_stable", :user_ssh_keys => ["2747877c-31d2-4683-9ac8-6030dc68f0d1"])
```

File: README.md [hunk 1]
```diff
@@ -93,8 +93,8 @@ Available parameters
 | spot_price_max | string | | No |
 | termination_time | string | | No |
 | tags | string | | No |
-| project_ssh_keys | string | | No |
-| user_ssh_keys | string | | No |
+| project_ssh_keys | collections of strings | | No |
```
Comment (edevenport) [inline README.md]: `collection` should not be plural here.

File: README.md [hunk 2]
```diff
@@ -93,8 +93,8 @@ Available parameters
...
+| user_ssh_keys | collections of strings | | No |
```
Comment (edevenport) [inline README.md]: `collection` should not be plural here.

**Comment by edevenport** [comment]
One last request while we are updating the README; search for `blah` and replace it with a valid ID string. It kind of lacks a formal style we should covey in examples.

**Comment by jasmingacic** [comment]
@edevenport as requested

**Pull request merged**